matisse/src/main/java/com/zhihu/matisse/ui/MatisseActivity.java

```diff
@@ -193,10 +195,10 @@ protected void onActivityResult(int requestCode, int resultCode, Intent data) {
                 }
                 updateBottomToolbar();
             }
-        } else if (requestCode == REQUEST_CODE_CAPTURE) {
+        } else if (requestCode == REQUEST_CODE_CAPTURE_PHOTO) {
```

Comment:
Here should be `if (requestCode == REQUEST_CODE_CAPTURE_PHOTO || requestCode == REQUEST_CODE_CAPTURE_VIDEO)`